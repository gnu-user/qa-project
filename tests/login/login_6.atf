INVALID!!!
END________________________________000_000.00