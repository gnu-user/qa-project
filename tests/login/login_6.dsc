TITLE
    Corrupted available tickets file 
DESCRIPTION
    Attempt to initiate the login transaction by entering “login” into the 
    input, and then read in the current user accounts file. Enter a valid 
    username, and begin reading in an invalid available tickets file and verify 
    that an error occurs.
DEPENDENCIES
    login_5
