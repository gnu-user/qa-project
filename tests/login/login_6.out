Welcome.
> 
Enter username: 
Username accepted.
An error occurred reading the available tickets file, data corrupted.